mapclientplugins/geometryfitter/model/geometryfittermodel.py

```diff
@@ -101,15 +101,15 @@ def _initGraphicsModules(self):
         defaultTessellation = tessellationmodule.getDefaultTessellation()
         defaultTessellation.setRefinementFactors([12])
 
-    def _getFitSettingsFileName(self):
+    def getJsonSettingsFilename(self):
         return self._location + "-settings.json"
 
-    def _getDisplaySettingsFileName(self):
+    def getJsonDisplaySettingsFilename(self):
         return self._location + "-display-settings.json"
```

Comment:
If these methods aren't static then the initialisation shouldn't load the models.  This should be done separately only when needed.